index.html

```diff
@@ -43,5 +43,162 @@ <h2 class="heading-2">I am Software Developer</h2>
+                
+                  <div class="Stories-lang-article-1">
+                      <ul class="stories-lang-list">
+                          <li class="stories-list-article">html</li>
+                          <li class="stories-list-article">bootstrap</li>
+                          <li class="stories-list-article">Ruby</li>
+                      </ul>
+                  </div>
+            </div>
+        <button type="submit" class="button-articles">See Project</button>
+    
+            <div class="article-1">
+                <h4 class="heading-article"> Profesional Art Printing Data</h4>
+                  <p class="heading-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                
+                  <div class="Stories-lang-article-1">
+                      <ul class="stories-lang-list">
+                          <li class="stories-list-article">html</li>
+                          <li class="stories-list-article">bootstrap</li>
+                          <li class="stories-list-article">Ruby</li>
+                      </ul>
+                  </div>
+            </div>
+        <button type="submit" class="button-articles">See Project</button>
+    
+            <div class="article-1">
+                <h4 class="heading-article"> Profesional Art Printing Data</h4>
+                  <p class="heading-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                
+                  <div class="Stories-lang-article-1">
+                      <ul class="stories-lang-list">
+                          <li class="stories-list-article">html</li>
+                          <li class="stories-list-article">bootstrap</li>
+                          <li class="stories-list-article">Ruby</li>
+                      </ul>
+                  </div>
+            </div>
+            
+        <button type="submit" class="button-articles">See Project</button>
+    
+          </div> 
+        </section>
+    
+        <!-- About me -->
+
+        <section class="about-me-section">
+            <div class="about-title">
+                <h1>About me</h1>
+            </div>
+            <div class="about-para">
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div id="btn-line">
+                <button type="button" id="get-resume-btn">Get My Resume</button>
+            </div>
+            <div class="language">
+                <h1 class="language-header"><span><img src="./images/works/Rectangle-one.svg" alt=""></span>Language</h1>
```

Comment:
- [ ] My comment about `hx` tags applies to the following `h1` tags as well. I kindly invite you to address these issues.